src/pybind/mgr/dashboard/tox.ini

```diff
@@ -130,15 +130,20 @@ addopts =
 #    --aggressive
 #    --aggressive
 
+[isort]
+atomic = true
+multi_line_output = 2
+line_length = 100
+
 [testenv:fix]
 basepython=python3
 deps =
     {[base-lint]deps}
 commands =
     python --version
+    isort .
```

Comment:
We should add this to [testenv:lint]. This will check if there are any error catched by isort during lint.
```
    isort . --check
```